service/csv_service.go

```diff
@@ -0,0 +1,59 @@
+package service
+
+import (
+	"encoding/csv"
+	"io"
+	"os"
+	"errors"
+	"strconv"
+
+	"github.com/ErickRodriguezWize/academy-go-q42021/domain/model"
+)
+
+var (
+	CSVError = errors.New("Couldn't open CSV File")
+	ColumnParseError = errors.New("First Column of CSV most have an Integer Value")
+)
+
+
+//ReadCSV reads a .csv file specific path 
+func ReadCSV(path string, pkms *[]model.Pokemon) (error){
+	file, err := os.Open(path)
+	
+	if err != nil {
+		return  CSVError //errors.New("Couldn't Open CSV File")
```

Comment:
You should wrap your errors when these come from an external package.

This will be helpful when trying to debug the root cause of a bug.